numpy/lib/npyio.py

```diff
@@ -1303,6 +1303,13 @@ def loadtxt(fname, dtype=float, comments='#', delimiter=None,
     array([('alpha, #42', 10.), ('beta, #64',  2.)],
           dtype=[('label', '<U12'), ('value', '<f8')])
 
+    Quoted fields can be separated by multiple whitespace characters
+    >>> s = StringIO('"alpha, #42"       10.0\n"beta, #64" 2.0\n')
```

Comment:
Thanks for looking into this, this looks good to me and like a clear bugfix.  I am still travelling and not really around, so sorry for the small delay.

The one thing is that the circleCI runs are failing with:
```
/home/circleci/repo/venv/lib/python3.8/site-packages/numpy/__init__.py:docstring of numpy.loadtxt:253: ERROR: Unexpected indentation.
```

I think suspect should fix it though:
```suggestion

    >>> s = StringIO('"alpha, #42"       10.0\n"beta, #64" 2.0\n')
```